app/views/pages/index.html.slim

```diff
@@ -0,0 +1,32 @@
+.row
+  .medium-9.medium-centered.columns
+    h2
+      = "Sport news"
+    br
+  .medium-7.columns
+    - @pages.each do |page|
+      h5
+        =(link_to page.title, page)
+      p
+        span
+          i.fi-torso
+            = page.user.full_name
+            ="                   "
+        span
+          i.fi-calendar
+            = page.created_at.strftime("%m/%d/%Y at %I:%M%p   ")
```

Comment:
.created_at.strftime("%m/%d/%Y at %I:%M%p   ") - it's very good example for what kind of function we prefer to move in decorator)